components/CandidateTestView/Instructions/Instructions.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import scss from "../Instructions/Instructions.module.scss";
+import {CandidateTestModel} from "../../../Models/CandidateTestModel";
+
+const InstructionsBlock = (testModel: CandidateTestModel): JSX.Element => (
+    <div>
+        <p className={scss.instructionList}>{testModel.brief}</p>
+        <h2>Example</h2>
+        <p className={scss.instructionList}>{testModel.example}</p>
+        <h2>Input/Output</h2>
+        <ul className={scss.instructionList}>
+            <li className={scss.instructionList}><strong>[execution time limit] {testModel.executionTimeLimit} (js)</strong></li>
+            <li className={scss.instructionList}><strong>[input] {testModel.inputType}</strong><br/>{testModel.inputDescription}</li>
+            <li className={scss.instructionList}><strong>[output] {testModel.outputType}</strong><br/>{testModel.outputDescription}</li>
+        </ul>
+        <h2>[JavaScript (ES6)] Syntax Tips</h2>
+        <p className={scss.instructionList}>{testModel.jsTips}</p>
```

Comment:
If we are going with this, then we need to carefully format it and think about the UX.
Its not in our designs, and isn't easy to slot in.

If we are going to present code... then we should format it properly.
At the very least we should be putting it inside `pre` and `code` tags.

And really I think we'd want to format it similar to the code editor.

I'd still recommend that we just remove this for now though. :)